components/portal/main/controllers.js

```diff
@@ -33,6 +33,26 @@ define(['angular', 'require'], function(angular, require) {
       layoutMode: 'list', // other option is 'widgets
     };
 
+    /**
+     * Set Document title.
+     */
```

Comment:
I see that we call updateTitle with no param later in this file.  Maybe a comment on what happens when no param is called?  What happens with the parameter?